examples/recipes/robot_display_emotions.py

```diff
@@ -0,0 +1,147 @@
+from pitop import Pitop, Camera
+from PIL import Image, ImageFont, ImageDraw
+from pitop.processing.algorithms.faces import (
+    FaceDetector,
+    EmotionDetector
+)
+from signal import pause
+from pitop.processing.utils.vision_functions import import_opencv
+
+
+cv2 = import_opencv()
+
+'''
+Designed for the Prax robot configuration, this example will use the camera module to detect face emotions.
+The emotion will be displayed on the miniscreen alongside a confidence level. Also displayed is a live 68-point facial
+landmark detection of the face being detected.
+
+To change emoji's, go here to see a list of unicodes: https://unicode.org/emoji/charts/full-emoji-list.html
+'''
+
+emoji_unicodes = {
+    "Happy": u"\U0001F642",
+    "Anger": u"\U0001F620",
+    "Surprise": u"\U0001F62E",
+    "Neutral": u"\U0001F610",
+    "Sad": u"\U0001F641",
+    "Disgust": u"\U0001F62C",
+    "Looking": u"\U0001F440"
+}
+
+
+text_font = ImageFont.truetype("VeraMono.ttf", size=12)
+emoji_font = ImageFont.truetype("Symbola.ttf", size=50, encoding='unic')
```

Comment:
@jcapona how can we add this font to the OS? It's a good one for displaying emoji's on the miniscreen

Reply:
Yeah I think when I get around to writing a Further challenge for Prax I will definitely make a draw emoji function so let's just add it in for now

Reply:
`Symbola_hint.ttf` works though. the package only installs 1 font file:
```
/usr/share/fonts/truetype/ancient-scripts/Symbola_hint.ttf
```